### PR TITLE
Add CTA for accelerators on quickstart pages

### PR DESCRIPTION
**Pull request by rlh1994**
Adds a CTA to point to the relevant accelerator on each package quickstart page, I also added a generic CTA to the high level quickstart page.

Currently the links are directly to the accelerator, not sure if we want them to instead point to the marketing page? e.g. https://snowplow.io/data-product-accelerators/advanced-analytics-mobile/

Page with CTA: https://deploy-preview-346--snowplow-docs.netlify.app/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/web/

### CODE REVIEW

**Comment by Craig-Mace** [comment]
All looks good to me. Tested the links are all working. Think keeping the links to be relevant to the specific DPA is best with the overall marketing link on the top page, is the right way to go.

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
 ---
 title: "E-commerce"
 sidebar_position: 105
+hide_title: true
 ---
+# E-commerce Quickstart
```
Comment (stanch) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md:6]: Why? If you want different sidebar label and title, you can use:
```
title: "E-commerce Quickstart"
sidebar_label: "E-commerce"
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+
+```mdx-code-block
+import Admonition from "@theme/Admonition";
+import {SnowplowIcon} from "@site/src/components/SnowplowIcon/SnowplowIcon";
+```
+
+<Admonition icon={SnowplowIcon()} title="Unleash the power of your behavioral data" type="tip">
```
Comment (stanch) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md:13]: These admonitions all have the same text, save for the name of the accelerator. Could you make this a component, or at least a reusable markdown block? (Check the telemetry one for an example of a reusable block with parameters.) That way we wouldn’t need to change this in 6 different places when needed.

File: src/css/custom.css [hunk 3]
```diff
@@ -348,3 +348,25 @@ abbr[data-title]:focus::after {
...
   padding: 3px 5px;
 }
+
+.callToAction{
+  display: inline-flex;
+  border-radius: 1vh;
+  justify-content: center;
+  padding: 0.5rem 0.5rem;
+  margin: 0.5rem auto;
+  width:fit-content ;
```
Comment (stanch) [inline src/css/custom.css:358]: formatting

File: src/css/custom.css [hunk 4]
```diff
@@ -348,3 +348,25 @@ abbr[data-title]:focus::after {
...
+  color: #FFFFFF;
+  text-decoration-line: none;
+}
+
+.callToAction:hover{
+  color: #FFFFFF;
+  text-decoration-color: #FFFFFF;
+}
+
+.colouredSvg{
```
Comment (stanch) [inline src/css/custom.css:369]: US spelling please

**Review by stanch — CHANGES_REQUESTED** [review]

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md (hunk 1 above)
Comment (rlh1994) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md:6]: TIL

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md (hunk 2 above)
Comment (rlh1994) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/index.md]: I have done my best

**Review by stanch — APPROVED** [review]

**Pull request merged**